src/modules/did/did.js

```diff
@@ -795,6 +796,47 @@ class DIDModule {
       }
     });
 
+    // Fetch BBS+ keys if needed
```

Comment:
what would happen if:
- user has three did keys before this change (1, 2 and 3)
- user adds a BBS key

what would the index of the BBS key be? assuming 4, just want to make sure.

if user deletes a key after, would the bbs key still be index 4 or would it go to 3? if it goes to 3, there may be problems

Reply:
> assuming 4, just want to make sure.

Yes.

> if user deletes a key after, would the bbs key still be index 4

Remains 4. Key ids don't change and they behaving in the same way for BBS+ as for other types.

Reply:
sounds good, just wanted to confirm